csrc/mmdeploy/preprocess/transform/resize.cpp

```diff
@@ -74,12 +74,12 @@ class Resize : public Transform {
         MMDEPLOY_DEBUG(
             "neither 'scale' or 'scale_factor' is provided in input value. "
             "'img_scale' will be used");
-        if (-1 == img_scale_[1]) {
+        if (-1 == img_scale_[0]) {
           if (w < h) {
-            dst_w = img_scale_[0];
+            dst_w = img_scale_[1];
             dst_h = dst_w * h / w;
           } else {
-            dst_h = img_scale_[0];
+            dst_h = img_scale_[1];
             dst_w = dst_h * w / h;
           }
```

Comment:
When export preprocess, it will set the last dim to -1.  May not need to change this. 

https://github.com/open-mmlab/mmdeploy/blob/07e5d2d30504f884a44022ceeed5a56eb91455cb/mmdeploy/codebase/mmpretrain/deploy/classification.py#L311-L315

Reply:
Yes, but `img_scale_ = {height, width};` makes the first dim -1.